plugins/woocommerce/tests/php/src/Internal/FraudProtection/AdminSettingsHandlerTest.php

```diff
@@ -0,0 +1,377 @@
+			->method( 'get_authorization_url' )
+			->willReturn( 'https://example.com/connect' );
+
+		// Capture output.
+		ob_start();
+		$this->sut->handle_output_jetpack_connection_field( array() );
+		$output = ob_get_clean();
+
+		// Should contain Connect button.
+		$this->assertStringContainsString( 'Connect to Jetpack', $output );
+		$this->assertStringContainsString( 'jetpack_connection_button', $output );
+		$this->assertStringContainsString( 'https://example.com/connect', $output );
+	}
+
+	/**
+	 * Test that handle_output_jetpack_connection_field shows connected status when connected.
+	 */
+	public function test_handle_output_jetpack_connection_field_shows_connected_status(): void {
+		// Enable fraud protection.
+		update_option( 'woocommerce_feature_fraud_protection_enabled', 'yes' );
+
+		$this->sut->register();
+
+		// Mock connection status - connected.
+		$this->connection_manager_mock->expects( $this->once() )
+			->method( 'get_connection_status' )
+			->willReturn(
+				array(
+					'connected'  => true,
+					'error'      => '',
+					'error_code' => '',
+					'blog_id'    => 12345,
+				)
+			);
+
+		// Should not call get_authorization_url when connected.
+		$this->connection_manager_mock->expects( $this->never() )
+			->method( 'get_authorization_url' );
+
+		// Capture output.
+		ob_start();
+		$this->sut->handle_output_jetpack_connection_field( array() );
+		$output = ob_get_clean();
+
+		// Should show connected status.
+		$this->assertStringContainsString( 'Connected to Jetpack', $output );
+		$this->assertStringContainsString( 'Site ID: 12345', $output );
+		$this->assertStringContainsString( 'dashicons-yes-alt', $output );
+	}
+
+	/**
+	 * Test that handle_output_jetpack_connection_field shows error when authorization URL fails.
+	 */
+	public function test_handle_output_jetpack_connection_field_shows_error_when_url_fails(): void {
+		// Enable fraud protection.
+		update_option( 'woocommerce_feature_fraud_protection_enabled', 'yes' );
+
+		$this->sut->register();
+
+		// Mock connection status - not connected.
+		$this->connection_manager_mock->expects( $this->once() )
+			->method( 'get_connection_status' )
+			->willReturn(
+				array(
+					'connected'  => false,
+					'error'      => 'Jetpack not available',
+					'error_code' => 'jetpack_not_available',
+					'blog_id'    => null,
+				)
+			);
+
+		// Mock authorization URL - returns null (failure).
+		$this->connection_manager_mock->expects( $this->once() )
+			->method( 'get_authorization_url' )
+			->willReturn( null );
+
+		// Capture output.
+		ob_start();
+		$this->sut->handle_output_jetpack_connection_field( array() );
+		$output = ob_get_clean();
+
+		// Should show error message.
+		$this->assertStringContainsString( 'Jetpack not available', $output );
+		$this->assertStringNotContainsString( 'Connect to Jetpack', $output );
+	}
+
+	/**
+	 * Test that handle_enqueue_admin_scripts only enqueues on features section.
```

Comment:
There are some tests that check `handle_enqueue_admin_scripts`, but that has been removed now that we're using a simple link.